### DISCUSSION TITLE
Normalization of subset data

### DISCUSSION THREAD

**Discussion by kyleyhchan** [satijalab/seurat#4678] [Q&A]
Hi all,
I am working on a set of immune cell data. I have already done the reduction and identify all the immune cell types. Now im going to subset the Fibroblast and do second round UMAP. In this case, do I still have to do another normalization after I subset the data, or I can just RunUMAP?

Thank you.

**Comment by timoast** [comment]
If you used log-normalization, the value for each cell does not depend on the other cells in the dataset, so you do not need to renormalize after subsetting since the values will be the same. We do recommend re-running `FindVariableFeatures()`, `ScaleData()`, and `RunPCA()` after subsetting the data so that the variable features, scaled expression, and PCA can be more relevant to the subset cells rather than the whole original population.

**Reply by Sahu-Mousumi** [reply]
I am doing a similar analysis. Do I have to re-run these fuctions (FindVariableFeatures(), ScaleData(), and RunPCA()) on integrated assay or on RNA?
